app/src/main/java/com/example/stromtracker/ui/geraete/GeraeteFragment.kt

```diff
@@ -89,7 +89,6 @@ class GeraeteFragment : Fragment() {
                     Log.d("TAGGeraete", geraete.toString())
```

Comment:
Log messages should not go into develop if not required explicitly.

Reply:
Is going to be overwritten by another merge from @hoffmatteo